tests/test_dataset_matching.py

```diff
@@ -18,18 +18,16 @@ def compute_r2(observed_scores: list[float], expected_scores: list[float]) -> fl
 
     # sum of squares total
     mean_expected_scores = sum(expected_scores) / len(expected_scores)
-    ss_total = 0
+    ss_total = 0.
```

Comment:
I didn't add the dot because the arguments' type hints are floats, but better to be safe.

Reply:
I changed it because `mypy` complained.